spec/lib/court_data_adaptor/query/defendant/by_name_spec.rb

```diff
@@ -29,7 +29,7 @@ def self.resource
 
     context 'with mocking' do
       let(:resource) { self.class.resource }
-      let(:resultset) { instance_double('ResultSet') }
+      let(:resultset) { instance_double('ResultSet') } # rubocop:disable RSpec/StringAsInstanceDoubleConstant
```

Comment:
Rubocos is complaining about this, let's do it later.
This tests also needs refactor, it uses too many stubbed methods.